pinot-core/src/test/java/org/apache/pinot/queries/TextSearchQueriesTest.java

```diff
@@ -1864,6 +1878,14 @@ public void testInterSegment() {
     query = "SELECT count(*) FROM MyTable WHERE TEXT_MATCH(SKILLS_TEXT_COL_DICT, 'a and or in the are')";
     testInterSegmentAggregationQueryHelper(query, 0);
 
+    // query with excluded stop-words. they should be indexed
+    query = "SELECT count(*) FROM MyTable WHERE TEXT_MATCH(SKILLS_TEXT_COL, 'IT')";
```

Comment:
Can we try phrase queries to mimic what we saw in the customer prod use case ?

Reply:
Done